tests/api2/test_zzzz_stig.py

```diff
@@ -325,3 +375,39 @@ def test_stig_usage_reporting_disabled(setup_stig):
 
     with pytest.raises(CallError, match='Network activity "Anonymous usage statistics" is disabled'):
         call("network.general.will_perform_activity", "usage")
+
+
+def test_stig_prevent_truecommand_enable(setup_stig):
+    ''' Wnen in GPOS STIG mode TrueCommand must remain disabled '''
+    assert setup_stig['aal'] == "LEVEL_2"
+
+    with pytest.raises(Verr, match='TrueCommand cannot be enabled under General Purpose OS STIG'):
+        call('truecommand.update', {'enabled': True, 'api_key': '1234567890-ABCDE'}, job=True)
+
+
+def test_stig_prevent_docker_apps_enable(setup_stig):
+    ''' Wnen in GPOS STIG mode Apps must remain disabled '''
+    assert setup_stig['aal'] == "LEVEL_2"
+
+    with another_pool() as docker_pool:
+        with pytest.raises(ValidationErrors, match='Apps cannot be enabled under General Purpose OS STIG'):
+            call('docker.update', {'pool': docker_pool['name']}, job=True)
```

Comment:
This is invalid because it's using `call` rather than a properly established WS session with 2FA credentials. Same for other cases.